### PR TITLE
Ci/new testnet

### PR DESCRIPTION
**Pull request by kerber0x**
## Description and Motivation

This PR adds the new narwhal-2 testnet data. Also improves the `load_env_variables` alias so that it prints what chain you just loaded. 

## Related Issues

---

## Checklist:

- [x] I have read [Migaloo's contribution guidelines](https://github.com/White-Whale-Defi-Platform/migaloo-core/blob/main/docs/CONTRIBUTING.md).
- [x] My pull request has a sound title and description (not something vague like `Update index.md`)
- [x] All existing and new tests are passing.
- [x] I updated/added relevant documentation.
- [x] The code is formatted properly `cargo fmt --all --`.
- [x] Clippy doesn't report any issues `cargo clippy -- -D warnings`.
- [x] I have regenerated the schemas if needed `cargo schema`.

### CODE REVIEW

File: justfile [hunk 1]
```diff
@@ -106,9 +106,13 @@ store CHAIN ARTIFACT='all':
 migrate CHAIN ARTIFACT='all':
   scripts/deployment/migrate_liquidity_hub.sh -c {{CHAIN}} -m {{ARTIFACT}}
 
+# Alias to the rename-a64-artifacts recipe.
+rename-artifacts:
+  @just rename-a64-artifacts
```
Comment (kerber0x) [inline justfile:111]: another alias

File: scripts/deployment/deploy_liquidity_hub.sh [hunk 2]
```diff
@@ -128,29 +128,22 @@ function init_whale_lair() {
 	# Prepare the instantiation message
 	unbonding_period=86400000000000    # default value is 14 days, in nanoseconds
 	growth_rate="0.000000064300411522" # this is the value when you interpolate the growth rate to 2X with 365 days of bonding
+	bonding_assets='[
+                   {"native_token": {"denom": "uwhale"}},
+                   {"native_token": {"denom": "factory/migaloo1dpx7ytug647wefe7ajxmg5ejt68gxcfvw35f4e/test"}}
```
Comment (kerber0x) [inline scripts/deployment/deploy_liquidity_hub.sh:133]: fixed the whale_lair init message on the deployment script

File: scripts/deployment/deploy_liquidity_hub.sh [hunk 3]
```diff
@@ -182,7 +175,7 @@ function init_incentive_factory() {
 	# Instantiate the contract
 	code_id=$(jq -r '.contracts[] | select (.wasm == "incentive_factory.wasm") | .code_id' $output_file)
 	$BINARY tx wasm instantiate $code_id "$init" --from $deployer --label "White Whale Incentive Factory" $TXFLAG --admin $deployer_address
-
+	sleep $tx_delay
```
Comment (kerber0x) [inline scripts/deployment/deploy_liquidity_hub.sh:178]: added delays in here so there's time for the node to catch the tx so the next query doesn't fail

**Review by kaimen-sano — APPROVED** [review]
Code changes LGTM, I haven't verified the contract address though.

**Comment by kerber0x** [comment]
> Code changes LGTM, I haven't verified the contract address though.

yeah `factory/migaloo1dpx7ytug647wefe7ajxmg5ejt68gxcfvw35f4e/test` is just an example, this is meant to be tweaked as needed.

**Pull request merged**